### ISSUE TITLE
Dependency required 'test-unit'

### ISSUE THREAD

**Issue by tomsquest** [ngauthier/hydra#35]
It seems that 'test-unit' is required as a dependency.
Otherwise, the message "no such file to load -- test/unit/testresult" is displayed.
Adding "gem 'test-unit' to my Gemfile solved the problem.

**Comment by mrmemes-eth** [comment]
Also experiencing this issue under ruby 1.9 (rails 3.0.5)

**Comment by pellegrino** [comment]
Same problem here (Ruby 1.9, rails 3.0.6)

**Comment by cookrn** [comment]
Also experiencing this in 1.9.2p180

**Comment by dedman** [comment]
Same here. 1.9.2p180, Rails 3.0.7

**Comment by bowsersenior** [comment]
Yes, this is a definite issue on Ruby 1.9. Ruby 1.8 includes `Test::Unit` in the core, but on 1.9 you need to install it as a gem.

I also strongly advise that `Hydra` start using a [Bundler-style gemspec and Gemfile](http://asciicasts.com/episodes/245-new-gem-with-bundler) to improve handling of dependencies. This will also make it much easier for developers to get up and running with `Hydra`.

**Comment by nguyenchiencong** [comment]
Definitely an issue, same pb for me:
- 1.9.2p180
- rails 3.1rc4

**Comment by bowsersenior** [comment]
Just noticed that hydra does not officially support ruby 1.9.2 yet:
- https://github.com/ngauthier/hydra/issues/29
